ocp-config/prov-app/cm.yml

```diff
@@ -138,12 +138,15 @@ objects:
       ods.image-tag=${ODS_IMAGE_TAG}
       ods.git-ref=${ODS_GIT_REF}
 
-      # createProjects configuration
-      jenkinspipeline.quickstarter.create-projects.desc=Create Project. Only used internally, therefore separated from other quickstarters
-      jenkinspipeline.quickstarter.create-projects.url=opendevstack/ods-core.git#${ODS_GIT_REF}/create-projects/Jenkinsfile
-      jenkinspipeline.quickstarter.create-projects.type=project
-
-      # Backend quickstarters
+      #Adminjob configuration: create projects, delete-projects and delete-components
+      jenkinspipeline.adminjobs.create-projects.desc=Create Project
+      jenkinspipeline.adminjobs.create-projects.url=opendevstack/ods-core.git#${ODS_GIT_REF}/create-projects/Jenkinsfile
+      jenkinspipeline.adminjobs.delete-projects.desc=Delete openshift project
+      jenkinspipeline.adminjobs.delete-projects.url=opendevstack/ods-core.git#${ODS_GIT_REF}/delete-projects/Jenkinsfile
+      jenkinspipeline.adminjobs.delete-components.desc=Delete openshift components
+      jenkinspipeline.adminjobs.delete-components.url=opendevstack/ods-core.git#${ODS_GIT_REF}/delete-components/Jenkinsfile
```

Comment:
Like the new name "adminjobs" instead of "quickstarters" 👍

Reply:
Thanks.